### ISSUE TITLE
Wrong shapes

### ISSUE THREAD

**Issue by resteqs** [rowak/nanoleaf-desktop#32]
My triangles are recognized as Hexagons and i have no possibility to change it. In the official app everything is fine so it has to be problem with this client (btw expect for this bug its great :D ) 

- [x] fix it :D

**Comment by rowak** [comment]
Do you have the original triangle lights ([Aurora](https://ca-media.nanoleaf.me/assets/img/products/compare/lp@2x.png)) or the new triangle lights ([Shapes Triangles](https://ca-media.nanoleaf.me/assets/img/products/compare/shapes-tri@2x.png))? The Aurora lights definitely shouldn't behave this way, but this Shapes Triangles are [unsupported](https://www.reddit.com/r/Nanoleaf/comments/nrqf69/nanoleaf_for_desktop_hexagons_update/h0pxzod?utm_source=share&utm_medium=web2x&context=3).

I don't currently know of a reliable way to identify the different Shapes devices, however, I can try whitelisting your device model as a bit of a workaround. Try running [this program](https://ufile.io/qba5c21u) (which outputs some basic device info) then copy the output it gives you and paste it here. You should see your device's name, model, and some other info.

**Comment by resteqs** [comment]
Yes I have the newer triangles. Correct me if I'm wrong, but wouldn't it be easier if after the "Hexagons" get recognized a pop up shows up where the user can choose what kind of panels he has? It's not an optimal solution for diffrent models connected together, but only a few people have it so it still would access a pretty big group of users.

**Comment by rowak** [comment]
You're right, the device selection popup is probably the next best way to do this, but I likely won't implement it unless I know for certain that it isn't possible to detect the device type based on its name and model. Did you try running that program I linked in my last comment? (https://ufile.io/qba5c21u) It gets the name and model from your device. If you provide your Shapes Triangles device name and model I can possibly add auto-detection in the app for it.

**Comment by resteqs** [comment]
No haven't yet. I will try later

**Comment by chn0x** [comment]
I also have the Shapes Triangles. I ran the program you provided, and it output this:

Name:  Shapes C83F
Model:  NL42
Manufacturer:  Nanoleaf
Firmware version:  6.1.2

**Comment by rowak** [comment]
Thanks! I'll whitelist the shapes triangles.

These are still unsupported in the app though, and I don't plan on adding support for them right now. Out of curiosity, have you tested any of the other functions in the app with the shapes triangles? Does anything else appear broken?

**Comment by rowak** [comment]
Hey, I whitelisted the shapes triangles, so they should be recognized by the app now. You can download an updated build [here](https://ufile.io/gcj892au).

**Comment by chn0x** [comment]
They are indeed recognized as triangles now. The preview shows one solid, unchanging colour instead of what the triangles are currently displaying, but I don't care about that. I tried out Basic Effects, Color Effects, and Rhythm Effects, and those worked as expected. The Control, Discovery, Ambient Lighting, and Spotify Visualizer work as expected save for the most recent comment I made on the other issue (#35).

Thank you.
